### ISSUE TITLE
gocryptfs.conf no such file or directory

### ISSUE THREAD

**Issue by pfaelzerchen** [rfjakob/gocryptfs#385]
I have some trouble with the reverse mode and did not find a suitable forum. Maybe I configured something terribly wrong, but have not found a clue in the documentation.

I am trying to mount a reverse mode crypted folder for rsyncing into a backup storage. Therefore i initialized the unencrypted folder with
`gocryptfs -init -reverse /plain`

Then a configured a mountpoint in /etc/fstab like
`/plain   /crypt   fuse./usr/bin/gocryptfs ro,nodev,nosuid,force_owner=1000:1000,allow_other,quiet,reverse,config=/plain/.gocryptfs.reverse.conf,passfile=somefile   0   0`

and mounted it with mount /crypt. force_owner should allow to make the backup without involving root.

So far the directory is mounted and I see encrypted paths and files, but one entry seems strange
`-????????? ? ?        ?          ?            ? gocryptfs.conf`

I can not open the file. When I try to test, if i could restore the crypted folder with `gocryptfs /crypt /test_plain` I get an error:

> Cannot open config file: open /crypt/gocryptfs.conf: no such file or directory

I tried this with both

> gocryptfs 1.4.3; go-fuse 0.0~git20171124.0.14c3015; 2018-02-05 go1.9.3

from Ubuntu 18.10 Repository and

> gocryptfs v1.7 without_openssl; go-fuse v1.0.0-13-g75aaa80; 2019-03-17 go1.12

So what was the (maybe obvious) mistake I did?

**Comment by rfjakob** [comment]
Can confirm this behavoir. This is a bug, `gocryptfs.conf` should not show up when you use the `config=` option.

**Comment by rfjakob** [comment]
So you triggered this bug by using `config=` to set the config file to what whould have been used by default, anyway. I have fixed this via https://github.com/rfjakob/gocryptfs/commit/fba9b2b9951c5e6f53727380d2dd33938cd1eb4d , however, I wonder what the intention of the `config=` was?

**Comment by pfaelzerchen** [comment]
Removing the `config=` option solved the problem. Thanks for the real fastand helpful response. 

`config=` was a remainder. When first trying to mount via fstab automatically at reboot, I recognized that gocryptfs insisted on a `gocryptfs.conf`, but only a `gocryptfs.reverse.conf` was present. Therefore I used `config=` to fix this, but noticed later on that I forgot the `reverse` option in the fstab - and forgot `config=`...
